cmd/scollector/collectors/exim_linux.go

```diff
@@ -0,0 +1,113 @@
+package collectors
+
+import (
+	"fmt"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+
+	"bufio"
+	"bytes"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+func init() {
+	collectors = append(collectors, &IntervalCollector{F: c_exim_mailq, Interval: time.Minute * 1})
+}
+
+func c_exim_mailq() (opentsdb.MultiDataPoint, error) {
+	var md opentsdb.MultiDataPoint
+	mailq := exec.Command("/usr/bin/mailq")
```

Comment:
Oh there it is. We should consider adding that functionality to the util package, then, which handles cleaning up and timeouts nicely.